imap_processing/cdf/utils.py

```diff
@@ -41,7 +41,7 @@ def calc_start_time(shcoarse_time: int):
 
 def write_cdf(
     data: xr.Dataset,
-    description: str = "",
+    descriptor: str = "sci",
```

Comment:
To you question, I think we shouldn't make this optional input now since this field is pretty generic and can be so different based on individual instrument. 

Or if we put it in dataset.attrs, then we can abstract from that too. But could be up for discussion.

Reply:
> Or if we put it in dataset.attrs, then we can abstract from that too. But could be up for discussion.

I like this idea, but I am also not sure how it would look because the instrument already has a "Descriptor" which is: "SWE>Solar Wind Electron" which isn't what we want unfortunately.

Reply:
OK, I just made it non-optional now. We can look into the attrs idea later.

Reply:
that sounds good. I can do that in follow-up PR.